### PR TITLE
ak branch test

### PR DESCRIPTION
**Pull request by helloimak**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { StyleSheet, Text, View } from 'react-native';
 export default function App() {
   return (
     <View style={styles.container}>
-      <Text>Tickle Ball</Text>
+      <Text>Tickle Ball Branch Test Ak</Text>
```
Comment (calvinwchung) [inline App.js]: Please use your full name

Comment (jslee423) [inline App.js]: it should be test jason

**Review by jslee423 — DISMISSED** [review]

**Review by calvinwchung — CHANGES_REQUESTED** [review]

**Review by calvinwchung — APPROVED** [review]

File: App.js (hunk 1 above)
Comment (helloimak) [reply App.js]: fixed

**Review by calvinwchung — APPROVED** [review]

**Review by calvinwchung — APPROVED** [review]

**Review by calvinwchung — APPROVED** [review]

**Review by calvinwchung — APPROVED** [review]

**Review by jslee423 — APPROVED** [review]
this shits wack

**Pull request merged**